src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,29 @@
+package core.basesyntax;
+
+public class Circle extends Figure {
+    private int radius;
+
+    public Circle(Color color, int radius) {
+        super(color);
+        this.radius = radius;
+    }
+
+    public int getRadius() {
+        return radius;
+    }
+
+    public void setRadius(int radius) {
+        this.radius = radius;
+    }
+
+    @Override
+    public double getArea() {
+        return Math.PI * getRadius() * getRadius();
+    }
+
+    @Override
+    public void draw() {
+        System.out.println("Figure: circle, area: " + getArea()
+                + " sq.units, radius: " + getRadius() + " units, color: " + getColor());
+    }
```

Comment:
```suggestion
        System.out.println("Figure: circle, area: " + getArea()
                + " sq.units, radius: " + radius + " units, color: " + getColor());
    }
```
you have direct access to this field - just access it directly